README.md

```diff
@@ -1,134 +1,174 @@
-* [Driving Enhancement](#-driving-enhancement)
+* [Driving Enhancements](#-driving-enhancements)
 * [Branch Definitions](#-branch-definitions)
-* [**✅Recommended Branches✅**](#-recommended-branches)
-* [How-Tos](#-How-Tos-)
-* [**💰Donate💰**](#-donate-)
-* [Pull Requests](#-Pull-Requests-)
+* [Recommended Branches](#-recommended-branches)
+* [How-To's](#-How-Tos)
+* [Donate](#-donate)
+* [Pull Requests](#-Pull-Requests)
 * [Special Thanks](#-special-thanks)
 * [Licensing](#licensing)
 
 ---
 
-💭 Join our Discord 💭
----
+<details><summary><h3>💭 Join our Discord</h3></summary>
 
-Join sunnyhaibin's sunnypilot server!
+ Join the official sunnypilot Discord server to stay up to date with all the latest features and be a part of shaping the future of sunnypilot!
 * https://discord.gg/wRW3meAgtx
+ 
+    ![Discord Shield](https://discordapp.com/api/guilds/880416502577266699/widget.png?style=shield)
+ 
+</details>
 
-🚨 Read Before Installing 🚨
----
-
-It is recommended to read this entire page before proceeding. This will ensure that you fully understand each added feature on sunnypilot and you are selecting the right branch for your car to have the best driving experience.
+<details><summary><h3>🚨 Read Before Installing</h3></summary>
+ 
+It is recommended to read this entire page before proceeding. This will ensure that you fully understand each added feature on sunnypilot, and you are selecting the right branch for your car (and device) to have the best driving experience.
 
-sunnypilot is recommended to be used for **most** of the following car makes' models:
+sunnypilot is compatible with **most** models of the following car manufacturers:
 * Hyundai/Kia/Genesis (**HKG**)
 * Honda/Acura
 * Toyota/Lexus
 * Subaru
 * Chrysler/Jeep/RAM (**FCA**)
 * General Motors (**GM**)
+* VW/Audi/Skoda
 
-It is a fork of [comma.ai's openpilot](https://github.com/commaai/openpilot). By installing this software, you accept all responsibility for anything that might occur while you use it. All contributors to sunnypilot are not liable. ❗<ins>**Use at your own risk.**</ins>❗
+This is a fork of [comma.ai's openpilot](https://github.com/commaai/openpilot). By installing this software, you accept all responsibility for anything that might occur while you use it. All contributors to sunnypilot are not liable. ❗<ins>**Use at your own risk.**</ins>❗
+</details>
 
-### Safety Modifications
-All [official sunnypilot branches from sunnypilot's official GitHub repository](https://github.com/sunnyhaibin/sunnypilot/branches) strictly adhere to comma.ai's safety policy.
+<details><summary><h3>⛔ Prohibited Safety Modifications</h3></summary>
+ 
+All [official sunnypilot branches](https://github.com/sunnyhaibin/sunnypilot/branches) strictly adhere to [comma.ai's safety policy](https://github.com/commaai/openpilot/blob/master/docs/SAFETY.md). Any changes to this policy will result in your fork and/or device being banned from both comma.ai and sunnypilot channels.
```

Comment:
```suggestion
All [official sunnypilot branches](https://github.com/sunnyhaibin/sunnypilot/branches) strictly adhere to [comma.ai's safety policy](https://github.com/commaai/openpilot/blob/master/docs/SAFETY.md). Any changes in the software that alter the legitimacy or enforcement of this policy may result in your fork and/or device being banned from both comma.ai and sunnypilot channels.
```
This may sound more explicit. What do you think?

Reply:
I do agree, how about something like:  

> Any changes to* the software that alter ~~the legitimacy or enforcement of~~ this policy may result in your fork and/or device being banned from both comma.ai and sunnypilot channels.

Less is more seen as we are referring to the policy in the sentence before?